### DISCUSSION TITLE
Helm values dremio deploy fail liveness, readness and startup probes on port 44349

### DISCUSSION THREAD

**Discussion by tiagosouzatfs** [artifacthub/hub#4134] [Q&A]
When i enable liveness and readness or startup on port 44349, or changing to any port ephemeral, the deploy fail. But, when i disable liveness and readness or startup probes the aplication dremio performs perfectly. What might be happening? Before that anyone ask, i increase a lot time into params of probes.

**Comment by cynthia-sg** [comment]
Hi @tiagosouzatfs  👋

For issues with the content listed on [artifacthub.io](http://artifacthub.io/) it's best to contact the particular project directly. In the links section of the right panel in the package view you usually may find some links that can point you to the project's repository or the maintainers emails.

Hope this helps 🙂
